dali/kernels/slice/slice_flip_normalize_permute_cpu.h

```diff
@@ -58,28 +58,30 @@ struct NormalizePolicy {
 };
 
 template <typename OutputType, int Dims>
-inline void ZeroPad(OutputType *output,
-                    TensorShape<Dims> out_strides,
-                    TensorShape<Dims> padded_out_shape,
-                    std::integral_constant<int, 1>) {
+inline void Pad(OutputType *output,
+                TensorShape<Dims> out_strides,
+                TensorShape<Dims> padded_out_shape,
+                float fill_value,
+                std::integral_constant<int, 1>) {
   constexpr auto d = Dims - 1;
   // zero pad
   for (int64_t i = 0; i < padded_out_shape[d]; i++) {
-    *output = 0;
+    *output = fill_value;
```

Comment:
```suggestion
   *output = static_cast<OutputType>(fill_value);
```
just like it is done below?

Reply:
...why not Convert<OutputType>?

Reply:
done